internal/controller/metrics.go

```diff
@@ -234,11 +255,31 @@ func DeleteWorkloadProtectionStatusMetric(labels prometheus.Labels) bool {
 	return workloadProtectionStatus.Delete(labels)
 }
 
+// drClusterAvailableStatus Metrics
+func DRClusterAvailableStatusLabels(drCluster *rmn.DRCluster, errorMessage string) prometheus.Labels {
+	return prometheus.Labels{
+		ObjType:      "DRCluster",
+		ObjName:      drCluster.Name,
+		ErrorMessage: errorMessage,
```

Comment:
I think this will create a new metric, i.e the same DRCluster will have a metric where the value for this label would be "<some error>" and another where the value for this label would be "". IOW, each unique value for the label is a new metric as such, as per my understanding. Could you check in tests and see if that is what happens?